### ISSUE TITLE
Don't make all our links blue

### ISSUE THREAD

**Issue by johncowen** [kumahq/kuma-gui#3260]
### What happened?

TLDR;

Remove blue styling from in-alert links

---

We've had discussions in the past about anchor link design treatment and I spotted this (https://github.com/kumahq/kuma-gui/pull/3249#issuecomment-2514346008) today and thought it would be a good opportunity to formalize the decision around that a little by writing it down in an issue. Trying to be a little jovial here 😅 , but essentially putting a big underline underneath the fact that we shouldn't always use blue styling for links

In the past we've had discussions on making sure that all our anchor links in our entire application are blue, because Internet.

An important thing is: Internet != Design i.e. there are other considerations to take into account when deciding these things.

In the above PR for example (a kongponents upgrade) the styling that the authors are trying to enforce is yellow link text on yellow background, because Design.

![390137300-f677d974-7e25-49e3-9b03-4a942e962cb5](https://github.com/user-attachments/assets/65572196-b369-4e5a-a7f7-f331425370fc)

Even with the above PR applied we still have:

![Screenshot 2024-12-03 at 12 08 26](https://github.com/user-attachments/assets/fe5107ba-70c3-4018-a361-2365b8423940)

**Comment by johncowen** [comment]
I just checked how other folks are using this, and it does seem like they also have yellow alerts with blue links in them even after https://github.com/Kong/kongponents/pull/2528. I guess we might want to keep things as is also 🤷 (i.e. blue with underline on hover)

**Comment by johncowen** [comment]
I noticed in https://github.com/kumahq/kuma-gui/pull/3308 that I'm about to merge that there was this change:

https://github.com/Kong/kongponents/commit/84a4e9f75c7bff9b136ae3e89d7cc2be51dbcc3f

That commit might help things here

**Comment by johncowen** [comment]
I noticed this seems to now be like this:

<img width="1049" alt="Screenshot 2024-12-19 at 11 52 08" src="https://github.com/user-attachments/assets/5724b7d0-3ff6-4e5b-ac83-31c1bf272d46" />

Note the difference in the screengrab from the description that the link is no longer blue. We haven't done anything here yet so I'm pretty sure this was a consequence of https://github.com/Kong/kongponents/commit/84a4e9f75c7bff9b136ae3e89d7cc2be51dbcc3f

Not sure whether we should close this issue or not 🤔 seeing as the above commit causes issues elsewhere:

- https://github.com/kumahq/kuma-gui/pull/3317
- https://github.com/kumahq/kuma-gui/pull/3323

(the above PRs fix the issues our side)

**Comment by lahabana** [comment]
We can't remember things about this. Let's reopen if design complains
